src/foundry/foundry.js/pixi/containers/canvasLayers/placeablesLayer.d.ts

```diff
@@ -268,6 +283,30 @@ declare global {
       options?: DocumentModificationContext
     ): Promise<Array<InstanceType<ConfiguredDocumentClassForName<DocumentName>>>>;
 
+    /**
+     * Get the world-transformed drop position.
+     * @returns Returns the transformed x, y co-ordinates, or false if the drag event was outside the canvas.
+     */
+    protected _canvasCoordinatesFromDrop(
+      event: DragEvent,
+      {
+        center
+      }: {
+        /**
+         * Return the co-ordinates of the center of the nearest grid element.
+         * @defaultValue `true`
+         */
+        center: boolean;
+      }
+    ): [tx: number, ty: number] | false;
```

Comment:
```suggestion
    protected _canvasCoordinatesFromDrop(
      event: DragEvent,
      {
        center
      }?: {
        /**
         * Return the co-ordinates of the center of the nearest grid element.
         * @defaultValue `true`
         */
        center?: boolean | undefined;
      } | undefined
    ): [tx: number, ty: number] | false;
```

Reply:
you could also consider extracting the object type as an interface `CanvasCoordinatesFromDropOptions`.